base/v0_2_exp/schema.go

```diff
@@ -70,13 +70,14 @@ type FileContents struct {
 }
 
 type Filesystem struct {
-	Device         string             `yaml:"device"`
-	Format         *string            `yaml:"format"`
-	Label          *string            `yaml:"label"`
-	Options        []FilesystemOption `yaml:"options"`
-	Path           *string            `yaml:"path"`
-	UUID           *string            `yaml:"uuid"`
-	WipeFilesystem *bool              `yaml:"wipe_filesystem"`
+	Device         string   `yaml:"device"`
+	Format         *string  `yaml:"format"`
+	Label          *string  `yaml:"label"`
+	Options        []string `yaml:"options"`
```

Comment:
Should probably note at least in the commit message why this is changing to a `[]string` from `[]FilesystemOption`, also after a quick grep that would leave the `FilesystemOption` type alias unused so it can probably be dropped.